app/controllers/comments_controller.rb

```diff
@@ -59,6 +65,11 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+      
+      format.js do 
+        render templat: "comments/destroy"
```

Comment:
You have a typo here. Rails is great at using conventional naming and inferring what you are trying to do, that is probably the reason why the code is still working. You can read more about it [here](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action).